elide-contrib/elide-dynamic-config-helpers/src/main/resources/templates/security.hbs

```diff
@@ -0,0 +1,19 @@
+package com.yahoo.elide.contrib.dynamicconfig.model;
+
+import java.security.Principal;
+import java.util.Optional;
+import com.yahoo.elide.annotation.SecurityCheck;
+import com.yahoo.elide.security.checks.prefab.Role.RoleMemberCheck;
+
+public class DynamicConfigOperationChecks {
```

Comment:
We need to remove the top level DynamicConfigOperationChecks and instead move PrincipalIs* Classes as the top level class. For ex. the class should be generated as
`package example;

import com.yahoo.elide.annotation.SecurityCheck;
import com.yahoo.elide.security.checks.prefab.Role.RoleMemberCheck;

@SecurityCheck(DynamicConfigOperationChecksPrincipalIsAdmin.PRINCIPAL_IS_ADMIN)
public class DynamicConfigOperationChecksPrincipalIsAdmin extends RoleMemberCheck {

	public static final String PRINCIPAL_IS_ADMIN = "Principal is Admin";

	public DynamicConfigOperationChecksPrincipalIsAdmin() {
		super("admin"); 

	}
}`

Reply:
We should return each roles as separate java class back to the caller program. This is to assist during the inmemory compile as we will not be able to parse the top level class to get the individual security class.